addons/cppcheckdata.py

```diff
@@ -689,7 +687,7 @@ def ArgumentParser():
     return parser
 
 
-def reportError(template, callstack=[], severity='', message='', id=''):
+def reportError(template, callstack=(), severity='', message='', id=''):
```

Comment:
user is supposed to provide a list in callstack

I am not python expert so is it better to have an empty tuple than empty list for some reason?